src/Resolution.ts

```diff
@@ -37,6 +38,8 @@ import {prepareAndValidateDomain} from './utils/prepareAndValidate';
 import {fromDecStringToHex} from './utils/namehash';
 import {UnsSupportedNetwork} from './types';
 
+const DEFAULT_UNS_PROXY_SERVICE_URL = 'https://resolve.unstoppabledomains.com';
```

Comment:
Shall we consider using the URL that have API Gateway?

Reply:
updated!